ui/src/components/VaultSummary.jsx

```diff
@@ -94,14 +105,22 @@ export function VaultSummary({ vault, brandToInfo, id }) {
   } = makeDisplayFunctions(brandToInfo);
 
   const {
-    debtSnapshot, // amount
-    interestRate, // ratio
-    liquidationRatio, // ratio
-    locked, // amount
-    status, // string
+    debtSnapshot,
+    asset,
+    interestRate,
+    liquidationRatio,
+    locked,
+    status,
   } = vault;
 
-  const debt = debtSnapshot?.debt;
+  const debt =
+    debtSnapshot &&
+    asset &&
```

Comment:
in what cases aren't these available? if not should the VaultSummary still be rendered?